pkgs/top-level/python-aliases.nix

```diff
@@ -36,4 +36,5 @@ mapAliases ({
   smart_open = smart-open; # added 2021-03-14
   google_api_python_client = google-api-python-client; # added 2021-03-19
   googleapis_common_protos = googleapis-common-protos; # added 2021-03-19
+  mechanicalsoup = MechanicalSoup;  # added 2021-06-01
```

Comment:
Isn't this backwards, how did this ever build? It shadows the actual `pythonPackages.mechanicalsoup` attr afaict